### DISCUSSION TITLE
How to use the logo better?

### DISCUSSION THREAD

**Discussion by adityaraute** [rbjks/rbjks.github.io#19] [Ideas]
![RBJKS logo](https://user-images.githubusercontent.com/43912470/221432167-6b292985-9a82-49c6-9f65-6f67363a212e.jpeg)

So here's a logo we have for the NGO. I'm not sure how to incorporate this logo well enough. Please suggest.

**Comment by agriyakhetarpal** [comment]
We could include the circle and the leaf graphics around it in a square while cropping out the text at the bottom

**Reply by adityaraute** [reply]
Hmm, that may work. Will we have to change the color scheme for the entire website though?

**Reply by agriyakhetarpal** [reply]
Not really, we can modify the logo if that is feasible at this point. Otherwise we should take a look at changing the colour scheme too because it doesn't look WCAG-compliant yet (the contrast ratio is low for some elements), but that's a topic for a different discussion thread. You can open an issue about that if you want or convert this to one

**Reply by adityaraute** [reply]
Yeah I think right now we should focus on modifying the logo and tackle the color scheme issue later. Would you like to work on the logo modifications?

**Reply by agriyakhetarpal** [reply]
Happy to do it, how would you suggest I start?

**Reply by adityaraute** [reply]
Modify the logo, name the present and modified asset files differently, then include the logo on the homepage navbar, footer, favicon - wherever you see fit

**Comment by adityaraute** [comment]
Creating an issue (#78 )  for this discussion to include the logo in our header and footer.
Still looking for more ideas.
